blade-graphics/src/vulkan/resource.rs

```diff
@@ -192,6 +294,14 @@ impl crate::traits::ResourceDevice for super::Context {
 
     fn create_buffer(&self, desc: crate::BufferDesc) -> super::Buffer {
         use vk::BufferUsageFlags as Buf;
+        let external_next = match &desc.memory {
```

Comment:
Please don't match on a reference. Nothing wrong with it in the community at large, but I don't want this pattern used in my projects.

Reply:
It relies on more sugar in the syntax without clear benefits. When looking at the match statement, I want to know exactly what's referenced and what's copied. Without the sugar, I can look at the statement, look at the type definition, and get that answer. With sugar, I need to consider more things.